apis/extension/device_share.go

```diff
@@ -89,6 +89,7 @@ type DeviceAllocation struct {
 	Minor     int32                      `json:"minor"`
 	Resources corev1.ResourceList        `json:"resources"`
 	Extension *DeviceAllocationExtension `json:"extension,omitempty"`
+	BusID     string                     `json:"busID,omitempty"`
```

Comment:
I prefer define it as this:
```
type DeviceAllocation struct {
	Minor     int32                      `json:"minor"`
	Resources corev1.ResourceList        `json:"resources"`
	Extension *DeviceAllocationExtension `json:"extension,omitempty"`
	Topology  *DeviceTopology            `json:"topology,omitempty"`
}

type DeviceTopology struct {
	// BusID is the domain:bus:device.function formatted identifier of PCI/PCIE device
	BusID string `json:"busID,omitempty"`
}
```

Reply:
OK！BusId is one of the attributes of the topology structure and can be encapsulated in the topology structure！